src/packages/client/src/generation/TSClient/Model.ts

```diff
@@ -138,7 +138,7 @@ ${indent(
           if (f.outputType.location === 'outputObjectTypes') {
             return `${f.name}?: ${getAggregateInputType(
               (f.outputType.type as DMMF.OutputType).name,
-            )}${f.name === 'count' ? ' | true' : ''}`
+            )}${f.name === '_count' || f.name === 'count' ? ' | true' : ''}`
```

Comment:
I am unsure what this code does exactly, however, because I see `groupByType` above, I just want to note that groupBy does not have the legacy non-underscore fields anymore, so looking for "count" may be incorrect.

Reply:
Nice catch